### PR TITLE
Python 3.4/5/6 and PyPy support.

### PR DESCRIPTION
**Pull request by jamadden**
- Update travis and include coverage
  - Reach 100% coverage (there's not much here)
- Add tox.ini
- Drop some umbrella deps and test deps in favor of being more explicit
- Rename .txt -> .rst

### CODE REVIEW

File: src/zope/app/session/http.py [hunk 1]
```diff
@@ -13,13 +13,11 @@
...
 """
 
 # BBB
 from zope.session.http import (ICookieClientIdManager,
                                MissingClientIdException,
                                notifyVirtualHostChanged,
                                CookieClientIdManager,
-                               digestEncode,
-                               cookieSafeTrans)
+                               digestEncode)
```
Comment (mgedmin) [inline src/zope/app/session/http.py:23]: I'm guessing `cookieSafeTrans` disappeared from zope.session.

**Review by mgedmin — APPROVED** [review]
~~tl;dr~~ lgtm

**Comment by mgedmin** [comment]
You have PyPI rights now.

File: src/zope/app/session/http.py (hunk 1 above)
Comment (jamadden) [reply src/zope/app/session/http.py:23]: Yes, it's gone. So this module was not importable.

**Comment by jamadden** [comment]
Thank you!

**Pull request merged**